src/components/CollectionDetailPage/CollectionMenu/CollectionMenu.tsx

```diff
@@ -125,6 +125,8 @@ export default class CollectionMenu extends React.PureComponent<Props> {
             inverted
             flowing
           />
+
+          {isCommitteeMember && collection.isPublished ? <Dropdown.Item text={t('collection_menu.jump_in')} /> : null}
```

Comment:
Based on what we talked about offline, we will allow this also if the collection is not published.

Reply:
If it is the owner of the collection or manager can also see this button. Remember that the committee has no access to this view.

Reply:
This button does nothing after being clicked